DBMS/DBMS4/assignment-1.sql

```diff
@@ -0,0 +1,49 @@
+-- to get all the member name whose category is same as category of KESHAV SHARMA
+SELECT member_name FROM members WHERE category = (SELECT category FROM members WHERE member_name = 'KESHAV SHARMA');
+
+
+
+-- SELECT command to display information on the books
+-- that have not been returned till date. The information should
+-- include book issue date, title, member name and due date.
+USE library;
+SELECT bi.issue_date, m.member_name,bi.due_date,t.title_name
+FROM book_issue bi,book_return br,members m,titles t,books b
+WHERE m.member_id=bi.member_id 
+AND b.title_id=t.title_id 
+AND(bi.member_id,bi.issue_date,bi.accession_no) 
+NOT IN (
+SELECT member_id,issue_date,accession_no 
+FROM book_return br 
+GROUP BY br.member_id,br.accession_no,br.issue_date)
+GROUP BY bi.member_id,bi.accession_no,bi.issue_date; 
+
+
+
+
+-- SELECT command to display information on the books
+-- that have been returned after their due dates. The information
+-- should include book issue date, title, member name and due date.
+SELECT br.issue_date, m.member_name,bi.due_date,t.title_name
+FROM book_issue bi,book_return br,members m,titles t,books b
```

Comment:
ACTION: book_return br is not required as issue_date can befetched by book_issue bi